packages/cli/scripts/bootstrapFinder.ts

```diff
@@ -0,0 +1,84 @@
+import jayson from 'jayson/promise/index.js'
+
+import { createBeaconConfig, defaultChainConfig, BeaconConfig } from '@lodestar/config'
+import { genesisData } from '@lodestar/config/networks'
+import { BeaconLightClientNetworkContentType, fromHexString, getBeaconContentKey, LightClientBootstrapKey, LightClientOptimisticUpdateKey, LightClientUpdatesByRange, LightClientUpdatesByRangeKey, ProtocolId, toHexString } from 'portalnetwork'
+
+import { ssz } from '@lodestar/types'
+import { ForkName } from '@lodestar/params'
+import { computeSyncPeriodAtSlot } from '@lodestar/light-client/utils'
+import { concatBytes, hexToBytes } from '@ethereumjs/util'
+
+
+const { Client } = jayson
+
+const main = async () => {
+    const beaconConfig = createBeaconConfig(defaultChainConfig, hexToBytes(genesisData.mainnet.genesisValidatorsRoot))
+    const capellaForkDigest = beaconConfig.forkName2ForkDigest(ForkName.capella)
```

Comment:
you can make this more fork agnostic by getting `const fork = config.getForkName(slot)`... but you will anyway get the corresponding `version` (fork) in the beacon api responses and you can use that to create forkdigest for pushing data in portal network